dev/package-examples/endpoint-0.0.1/dataset/endpoint.events/manifest.yml

```diff
@@ -0,0 +1,6 @@
+title: Endpoint Events
```

Comment:
Not for this file, but for the path. I think this should be just `events` and not `endpoint.events`. I assume the problem you tried to solve with this is solved in https://github.com/elastic/package-registry/pull/176

Reply:
Yeah I was going off the code that's currently in `feature-ingest` https://github.com/elastic/kibana/blob/feature-ingest/x-pack/plugins/ingest_manager/server/services/epm/elasticsearch/index.ts#L13

So if I understand issue https://github.com/elastic/package-registry/pull/176 correctly there's going to be a field in the manifest.yml of a dataset and I can set the `path` there? and then the code I linked above will look for `dataset.path` instead of doing `${dataset.type}-${dataset.name}` ?

I think the `path` would be `endpoint.events`?

Reply:
If you call the directory `endpoint` you should not have to do anything as by default it assume path and id are identical. In this case the `dataset.id == endpoint`. The type of the data is defined as `events` so you should get directly `events-endpoint-*`.

Reply:
ah ok, yeah I think the reason I chose `endpoint.events` is because I was worried that someone looking at the directories wouldn't immediately know what `dataset/endpoint` but I also didn't want to do `dataset/events` because the name would be `events-events`. I'll change it to `endpoint` and if people are confused we can change it later.